nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyArrayTests.java

```diff
@@ -0,0 +1,91 @@
+package com.uber.nullaway;
+
+import com.google.errorprone.CompilationTestHelper;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class NullAwayJSpecifyArrayTests extends NullAwayTestsBase {
+
+  @Test
+  public void ArrayTypeAnnotationDereference() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  static @Nullable Integer [] fizz = {1};",
+            "  static void foo() {",
+            "  // OK: @Nullable annotations on type are ignored currently.",
+            "  // however, this should report an error eventually.",
+            "  int bar = fizz.length;",
```

Comment:
I disagree, there should be no error here.  Am I missing something?

Reply:
Also, please indent the test code within the strings correctly, to make things more readable